explorer/rest/db/DatabaseConnection.py

```diff
@@ -0,0 +1,66 @@
+from psycopg2 import pool
+
+
+class DatabaseConnectionPool:
+	"""Database connection pool class."""
+
+	def __init__(self, db_config, min_connections=1, max_connections=10):
+		"""
+		Initialize the database connection pool with given configurations.
+
+		:param db_config: Dictionary containing connection details.
+		:param min_connections: Minimum number of connections in the pool. Default is 1.
+		:param max_connections: Maximum number of connections in the pool. Default is 10.
```

Comment:
i don't think we need parameter documentation here and would prefer keeping documentation style consistent with sdk documentation